File: Python/tests/test_dd_hyperbolic_single.py
import pytest

from hbayesdm.models import dd_hyperbolic_single


def test_dd_hyperbolic_single():
    _ = dd_hyperbolic_single(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_dd_hyperbolic_single.R
context("Test dd_hyperbolic_single")
library(hBayesDM)

test_that("Test dd_hyperbolic_single", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(dd_hyperbolic_single(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
